<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QTabWidget" name="mainTabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>800</width>
     <height>470</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="configTab">
    <attribute name="title">
     <string>Configuration</string>
    </attribute>
    <widget class="QPushButton" name="simulationButton">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>131</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Simulation</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="mappingTab">
    <attribute name="title">
     <string>Mapping</string>
    </attribute>
    <widget class="QGraphicsView" name="mappingGraphicsView">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>0</y>
       <width>770</width>
       <height>430</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="debugTab">
    <attribute name="title">
     <string>Debug</string>
    </attribute>
    <widget class="QTextEdit" name="debugTextEdit">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>0</y>
       <width>770</width>
       <height>430</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Monaco</family>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="manualTab">
    <attribute name="title">
     <string>Manual Control</string>
    </attribute>
    <widget class="QGroupBox" name="directionGroupBox">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>100</y>
       <width>300</width>
       <height>240</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Direction</string>
     </property>
     <widget class="QPushButton" name="upButton">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>50</y>
        <width>80</width>
        <height>80</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>48</pointsize>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
      <property name="text">
       <string>⇧</string>
      </property>
     </widget>
     <widget class="QPushButton" name="downButton">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>140</y>
        <width>80</width>
        <height>80</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>48</pointsize>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
      <property name="text">
       <string>⇩</string>
      </property>
     </widget>
     <widget class="QPushButton" name="leftButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>80</width>
        <height>80</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>48</pointsize>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
      <property name="text">
       <string>⇦</string>
      </property>
     </widget>
     <widget class="QPushButton" name="rightButton">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>140</y>
        <width>80</width>
        <height>80</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>48</pointsize>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
      <property name="text">
       <string>⇨</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="gearGroupBox">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>100</y>
       <width>150</width>
       <height>240</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Gear</string>
     </property>
     <widget class="QLCDNumber" name="gearLcdNumber">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>90</width>
        <height>120</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="digitCount">
       <number>1</number>
      </property>
     </widget>
     <widget class="QSlider" name="gearSlider">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>190</y>
        <width>110</width>
        <height>20</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="brakeGroupBox">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>100</y>
       <width>170</width>
       <height>240</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Brake</string>
     </property>
     <widget class="QPushButton" name="brakeButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>130</width>
        <height>170</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>96</pointsize>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
      <property name="text">
       <string>℗</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
